src/Http/Requests/PaginationRequest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class PaginationRequest extends FormRequest
```

Comment:
Are you sure that's a good idea? What if you would like to add pagination to some controller's method what would require other form request? And do you really think that invalid pagination limit should crash application?